core/src/main/java/com/sequenceiq/cloudbreak/core/flow/ClusterBootstrapper.java

```diff
@@ -95,11 +102,15 @@ public void bootstrapNewNodes(StackScalingContext stackScalingContext) throws Cl
         try {
             ContainerOrchestrator containerOrchestrator = containerOrchestrators.get(containerOrchestratorTool);
             containerOrchestrator.bootstrapNewNodes(gatewayInstance.getPublicIp(), nodes);
-            clusterAvailabilityPollingService.pollWithTimeout(
-                    clusterAvailabilityCheckerTask,
-                    new ContainerOrchestratorClusterContext(stack, containerOrchestrator, gatewayInstance.getPublicIp(), nodes),
-                    POLLING_INTERVAL,
-                    MAX_POLLING_ATTEMPTS);
+            try {
+                clusterAvailabilityPollingService.pollWithTimeout(
+                        clusterAvailabilityCheckerTask,
+                        new ContainerOrchestratorClusterContext(stack, containerOrchestrator, gatewayInstance.getPublicIp(), nodes),
+                        POLLING_INTERVAL,
+                        MAX_POLLING_ATTEMPTS);
+            } catch (Exception e) {
+                clusterBootstrapperErrorHandler.handleError(containerOrchestrator, stack, nodes);
+            }
```

Comment:
The same as above.
The clusterAvailabilityCheckerTask can throw an exception only in case of a timeout:
- all other exceptions are handled inside ContainerOrchestrator.areAllNodesAvailable
- it would be better not to throw an exception, that's why we have PollingResult.TIMEOUT
- inner try-catch block won't be necessary at all